scripts/run-webservice-script.sh

```diff
@@ -3,10 +3,12 @@ set -o errexit
 set -o pipefail
 set -o nounset
 set -o xtrace
-# Uncomment this to use the actual Dockstore webservice from the package.json
-# JAR_PATH="https://artifacts.oicr.on.ca/artifactory/collab-release/io/dockstore/dockstore-webservice/${npm_package_config_webservice_version}/dockstore-webservice-${npm_package_config_webservice_version}.jar"
-# Uncomment this to use the CircleCI jar
-JAR_PATH="https://12143-33383826-gh.circle-artifacts.com/0/tmp/artifacts/dockstore-webservice-1.12.0-alpha.1-SNAPSHOT.jar"
+if [ "$npm_package_config_use_circle" = true ]
+then
+	JAR_PATH="https://""${npm_package_config_circle_build_id}""-33383826-gh.circle-artifacts.com/0/tmp/artifacts/dockstore-webservice-1.12.0-alpha.1-SNAPSHOT.jar"
```

Comment:
Could you have `1.12.0-alpha.1-SNAPSHOT.jar` come from the package.json as well? Then we'd only have to change one file whenever we need to switch between release/CircleCI.

Reply:
I noticed that. but it kind of infrequently changes so it's not too much of a deal. The plan I had if we really wanted to change that is to simply change the upload file name to not have the version name in it.

Reply:
This isn't on artifactory, so I'd be fine with changing the upload file name to circle